pallets/connectors/Cargo.toml

```diff
@@ -16,6 +16,7 @@ ethabi = { version = "16.0", default-features = false }
 frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 frame-system = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 hex = { version = "0.4.3", default_features = false }
+orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", default-features = false, branch = "polkadot-v0.9.29" }
```

Comment:
No `std` feature on traits?

Reply:
It was already in `std` since it was already a `[dev.dependencies]` :)